static/src/stylesheets/module/atoms/_storyquestions.scss

```diff
@@ -0,0 +1,139 @@
+.user__question {
+    padding-bottom: $gs-baseline;
+    overflow: auto;
+}
+.user__question-title {
+    font-weight: 900;
+    font-size: 22px;
+    line-height: 24px;
+    color: $news-main-1;
+    font-family: $f-serif-headline;
+
+    @include mq($from: tablet) {
+        font-size: 26px;
+        line-height: 28px;
+    }
+
+    span {
+        display: block;
+    }
+}
+
+.user__question-title--secondary {
+    color: $neutral-1;
+}
+
+.user__questions-text--wrapper {
+    position: relative;
+    overflow: hidden;
+
+    &:hover {
+        .user__question-text {
+            color: #ffffff;
+
+            span {
+                background-color: $news-main-1;
+            }
+        }
+        .tick {
+            background-color: $news-main-1;
+
+            svg {
```

Comment:
Best to add a more specific class to the svg. It will be added to the span and then the svg will be given the same class with `__svg` after it. For example, if the span has the class `user__tick-icon` or something similar, then the class on the svg would be `user__tick-icon__svg`. It is good to avoid nesting in css